components/financial_assistance/app/views/financial_assistance/deductions/_deduction.html.erb

```diff
@@ -1,30 +1,62 @@
+      <div class="d-flex col-sm p-0 mb-2">
+        <div class="mr-auto col-sm col-md-6 col-lg-3 p-0">
+          <label for="amount-<%= deduction.id %>" class="required"><%= l10n("faa.review.income.table.amount") %></label>
+          <input class="fake-disabled-input" id="amount-<%= deduction.id %>" disabled="true" value="<%= number_to_currency(deduction.amount) %>">
+        </div>
+        <div class="mr-auto col-sm col-md-6 col-lg-3 p-0">
+          <label for="frequency-<%= deduction.id %>" class="required"><%= l10n("faa.deductions.deduction_form.how_often") %></label>
+          <input class="fake-disabled-input" id="frequency-<%= deduction.id %>" disabled="true" value="<%= frequency_kind_options.invert[deduction.frequency_kind] %>">
+        </div>
+        <div class="mr-auto col-sm col-md-6 col-lg-3 p-0">
+          <label for="start-<%= deduction.id %>" class="required"><%= l10n("start_date") %></label>
+          <input class="fake-disabled-input" id="start-<%= deduction.id %>" disabled="true" value="<%= deduction.start_on %>">
+        </div>
+        <div class="mr-auto col-sm col-md-6 col-lg-3 p-0">
+          <label for="end-<%= deduction.id %>"><%= l10n("end_date") %></label>
+          <input class="fake-disabled-input" id="end-<%= deduction.id %>" disabled="true" value="<%= deduction.end_on.present? ? deduction.end_on : 'N/A' %>">
+        </div>
       </div>
-      <div class="col-md-3  no-pd form-group-lg no-pd class-fa-household">
-        <label class="static_label label-floatlabel">HOW OFTEN *</label>
-        <span class="field_value floatlabel form-control active-floatlabel" id="name"><%=  frequency_kind_options.invert[deduction.frequency_kind] %></span>
-      </div>
-      <div class="col-md-3  no-pd form-group-lg no-pd class-fa-household">
-        <label class="static_label label-floatlabel">START *</label>
-        <span class="field_value floatlabel form-control active-floatlabel" id="name"><%= deduction.start_on %></span>
-      </div>
-      <div class="col-md-2  no-pd form-group-lg no-pd class-fa-household">
-        <label class="static_label label-floatlabel">END</label>
-        <span class="field_value floatlabel form-control active-floatlabel" id="name"><%= deduction.end_on %></span>
-      </div>
-      <div class="col-md-1 no-pd form-group-lg class-fa-household fa-adjustment">
-          <a tabindex="0" onkeydown="handleButtonKeyDown(event, 'income-deduction-edit-<%= dom_id deduction %>')" id="income-deduction-edit-<%= dom_id deduction %>" class="deduction-edit edit-pr"><i class="fa fa-pencil fa-lg" aria-hidden="true"></i><span class="hide"><%=l10n("edit")%></span></a>
-          <a tabindex="0" onkeydown="handleButtonKeyDown(event, 'income-deduction-delete-<%= dom_id deduction %>')" id="income-deduction-delete-<%= dom_id deduction %>" class="deduction-delete"><i class="fa fa-trash-o fa-lg" aria-hidden="true"></i><span class="hide"><%=l10n("delete")%></span></a>
+      <div class="d-flex mb-2 mx-1">
```

Comment:
the mx-1 leaves a couple pixel gap when between the input row above and the buttons. Either remove left margin OR add to the input row above.